src/context.rs

```diff
@@ -575,21 +691,27 @@ impl OfflineAudioContext {
         Self {
             base,
             length,
-            render,
+            renderer,
         }
     }
 
+    /// `OfflineAudioContext` doesn't start rendering automatically
+    /// You need to call this function to start the audio rendering
     pub fn start_rendering(&mut self) -> AudioBuffer {
         // make buffer_size always a multiple of BUFFER_SIZE, so we can still render piecewise with
```

Comment:
I think actually this is no longer needed, now we can handle varying buffer lengths.
We can remove this code in another PR